Core/HLE/sceKernelInterrupt.cpp

```diff
@@ -826,6 +826,63 @@ static u32 sysclib_strncpy(u32 dest, u32 src, u32 size) {
 	return hleLogSuccessX(SCEKERNEL, dest);
 }
 
+static u32 sysclib_strtol(u32 t, u32 endStringtemp, int base) {
+	ERROR_LOG(SCEKERNEL, "Unimp sysclib_strtol(%08x, %08x, %i)", t ,endStringtemp, base);
+	// base == 0 seems to be handled as base == 10.
+	if (base == 0){
```

Comment:
`strtol()` has a pretty defined function, and I'd assume the PSP's version follows it.  These funcs are probably just some stdc lib compiled into the kernel.
https://en.cppreference.com/w/cpp/string/byte/strtol

Most likely 0 means autodetect, which means "0123" would be octal, "0x123" would be hex, and "123" would be decimal.

I'd suggest we just use the native `strtol()`.

-[Unknown]

Reply:
I don't know that how to u32 endStringtemp change to char**

Reply:
if all the arguments have similar format to `strtol()` i think you just need to to do this:
```c++
return strtol(Memory::GetCharPointer(t), Memory::GetCharPointer(endStringtemp), base);
```